packages/rancher-logging/generated-changes/patch/Chart.yaml.patch

```diff
@@ -10,7 +10,7 @@
 +  catalog.cattle.io/namespace: cattle-logging-system
 +  catalog.cattle.io/permits-os: linux,windows
 +  catalog.cattle.io/provides-gvr: logging.banzaicloud.io.clusterflow/v1beta1
-+  catalog.cattle.io/rancher-version: '>= 2.8.0-0 < 2.9.0-0'
++  catalog.cattle.io/rancher-version: '>= 2.9.0-0 < 2.10.0-0'
```

Comment:
Branching steps are also done in https://github.com/rancher/charts/pull/3284. That PR is blocked so better to do it here anyway!